### ISSUE TITLE
Switch robot replies to emit to formatter

### ISSUE THREAD

**Issue by reicruz** [ibm-cloud-solutions/hubot-ibmcloud-utils#24]
Need to verify and fix tests in affected repositories.

**Comment by aeweidne** [comment]
yeah this would definitely be a major release.  this project can be changed and released as a new major and then we can update the dependency in each project as we fix the tests
